packages/react-core/src/components/DescriptionList/DescriptionList.tsx

```diff
@@ -22,26 +32,44 @@ export interface DescriptionListProps extends Omit<React.HTMLProps<HTMLDListElem
     xl?: '1Col' | '2Col' | '3Col';
     '2xl'?: '1Col' | '2Col' | '3Col';
   };
+  autoFitModifier?: AutoFitModifiers;
```

Comment:
Could you add a description for this prop? The docs only show `AutoFitModifiers` as the type so it's a little unclear what should be passed in without looking at the code for the interface.

Or @redallen is it possible to get the exported interface to show on our docs page?

Reply:
Unfortunately not unless it's inlined. Can you inline the interface here @boaz0 ?

Reply:
Sure. I forgot that the docs won't show the type well unless it's inline. :smiley_cat: